src/app/centers/individual-collection-sheet/individual-collection-sheet.component.ts

```diff
@@ -0,0 +1,41 @@
+/** Angular imports */
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+
+/**
+ * Individual collection sheet component.
+ */
+@Component({
+  selector: 'mifosx-individual-collection-sheet',
+  templateUrl: './individual-collection-sheet.component.html',
+  styleUrls: ['./individual-collection-sheet.component.scss']
+})
+export class IndividualCollectionSheetComponent implements OnInit {
+    /** Collection sheet form. */
+    collectionSheetForm: FormGroup;
+    /** Branch Office data. */
+    branchOffice: string;
+    /** Repayment date. */
+    repaymentDate: Date;
+    /** Staff data. */
+    staff: string;
+
+  constructor(private formBuilder: FormBuilder) { }
+
+  createCollectionSheetForm() {
+    this.collectionSheetForm = this.formBuilder.group({
+      branchOffice: ['', Validators.required],
+      repaymentDate: ['', Validators.required],
+      staff: ['']
+    });
+  }
+
+  ngOnInit() {
+    this.createCollectionSheetForm();
+  }
+
+  submit() {
```

Comment:
why have you not connected the component to API?

Reply:
I do not have access to the data that populates the tables yet. I have used dummy data to fill the tables for now. Should i use dummy data to connect to the API?